packages/docs/docs/bots/bots-in-production.md

```diff
@@ -275,6 +275,31 @@ npx medplum bot deploy my-first-bot-production
 
 This pattern is especially powerful when deploying Bots **as part of a CI pipeline**.
 
+## Configuring Bot Logging
+
+Bots can be run at a very high volume - for example as part of an ADT feed or when triggered by high-frequency messages. This can result in thousands of invocations per day, which can be overwhelming to track. In these cases, it may make sense to fine-tune the level of logging for your Bots.
+
+There are two ways to control Bot logging in Medplum: the type of event logged (i.e., success, failure, etc.) and where the event gets logged to.
+
+### Logging By Event Type
+
+You can choose to only log certain events using the `Bot.auditEventTrigger` field. This element represents the criteria for when an `AuditEvent` resource should be created and has four possible values.
+
+| Trigger     | Description                                                                                    |
+| ----------- | ---------------------------------------------------------------------------------------------- |
+| `always`    | An audit event is created and saved every time the bot runs. This is the default setting.      |
+| `never`     | An audit event will never be saved by the bot.                                                 |
+| `on-error`  | An audit event is created and saved only if the bot throws an error.                           |
+| `on-output` | An audit event is created and saved whenever the bot logs any output as part of its execution. |
+
+### Logging By Destination
+
+Logging can also be limited by setting the destination that your Bot logs to. This is done using the `Bot.auditEventDestination` field. This element can be set to either `resource` or `log`.
+
+By default, `auditEventDestination` is set to `resource`. This will create an `AuditEvent` resource in the main database, which can be accessed by Medplum's API and will be visibile in your app. However, the operation is slower and takes up more space. It is best practice to always start by using `resource` as it allows for easier testing and debugging of your Bot.
+
+Setting `auditEventDestination` to `log` will only output an `AuditEvent` in your existing enterprise logging infrastructure, such as [AWS Athena](/docs/self-hosting/aws-athena-guide), [Datadog](/docs/self-hosting/datadog), or others. This can make testing and debugging more difficult, but is quicker and useful for high-volume operations as it will not clog the feed in your app.
```

Comment:
> it will not clog the feed in your app

This is not the main benefit.

The more important benefit is that it will not fill up or slow down the main database

Reply:
I've changed this